env.example

```diff
@@ -35,6 +35,14 @@ LOG_LEVEL=debug
 # LOG_FILE is used to set a destination path to write logs. Works in production mode only.
 LOG_FILE=
 
+# Used to set if Pino should log are indexed to ElasticSearch, PINO_ELASTIC=1 to enable
+PINO_ELASTIC=
+# Following are used to set up Pino-Elastic
```

Comment:
I have no idea how to set any of these.  Should they be config options or just something we do in code?  I think the latter, unless you think we need to tweak them a lot?